protocols/request-response/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@
 
 - Update `rust-version` to reflect the actual MSRV: 1.62.0. See [PR 3090].
 
+- Rename types as per [discussion 2174].
```

Comment:
We released `llibp2p-request-response` `v0.23.0` with `libp2p` `v0.50.0`.

If I understand correctly, this pull request does introduce a breaking change. Thus, would you mind bumping the version in the `libp2p-request-reponse` `Cargo.toml` and all its dependents and adding a changelog entry?

Reply:
Thanks Max, updated